mysite/missions/templates/missions/main.html

```diff
@@ -115,6 +115,26 @@ <h3 class="up-top"><img alt='Training Missions' id='training-missions' alt='Trai
     {% include 'missions/mission_card.html' %}
     {% endwith %}
     {% endwith %}
+
+    <div class="clearer">&nbsp;</div>
+    </div>
+
+    <h2>Setup your Python environment</h2>
+
+    <p>These missions teach you how to install Python packages and create
+    virtual environments.</p>
+
+    <div class="submodule-container">
+    {% with fully_completed_missions.pipvirtualenv as mission_completed %}
+    {% with "Using pip & virtualenv" as mission_title %}
```

Comment:
That red highlighting on the ampersand (&) doesn't look too good! Do you need to escape that?